_resources/_styles/landingpage.styl

```diff
@@ -80,10 +80,10 @@ label {
   @media laptop {
     .LandingPageQuote {
       width: 75%;
-      margin: 8rem 4rem 0 4rem
+      margin: 8rem 4rem 0 4rem;
     }
   }
-.LandingPageQuoteSpan {
+.Italic {
```

Comment:
This isn't accessible now. Remember, we want the page to look usable and understandable without CSS. Use the <i> tag and declare it in the CSS instead of 'Italic'